src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -325,4 +326,88 @@ avoiding peak traffic in order to address both inconsistencies diagnostic and fi
 
 #### How to solve
 
-Under development: Task for solving mailbox inconsistencies
\ No newline at end of file
+Under development: Task for solving mailbox inconsistencies
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serves multiple applications including James, the keyspaces need isolation, 
+that also means, by the credential provided for a James server, 
+you cannot use it to access or modify other keyspaces. 
+
+We recommend you to not use the initial admin user of Cassandra and provide 
+a different one with a minimized permissions for each application. 
+
+### Prerequisites
+
+We're gonna use the Cassandra super users to create roles and grant permissions for them. 
+To do that, Cassandra requires you to login via username/password authentication 
+and enable granting in cassandra configuration file.
+
+For example:
+```
+echo -e "\nauthenticator: PasswordAuthenticator" >> /etc/cassandra/cassandra.yaml
+echo -e "\nauthorizer: org.apache.cassandra.auth.CassandraAuthorizer" >> /etc/cassandra/cassandra.yaml
+```
+
+### Prepare Cassandra roles & keyspaces for James  
+
+#### Create a role
+
+Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/security.html) section `CREATE ROLE` for more information
+
+E.g.
+```
+CREATE ROLE linagora WITH PASSWORD = 'linagora' AND LOGIN = true;
+```
+#### Create a keyspace
+
+Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/ddl.html) section `CREATE KEYSPACE` for more information
+
+#### Grant permissions on created keyspace to the role
+
+The role to be used by James needs to have full rights on the keyspace 
+that James is using. Assuming the keyspace name is `linagora_keyspace` 
+and the role be `linagora`.
+
+```
+GRANT CREATE ON KEYSPACE linagora_keyspace TO linagora; // Permission to create tables on the appointed keyspace
+GRANT SELECT ON	KEYSPACE linagora_keyspace TO linagora; // Permission to select from tables on the appointed keyspace
+GRANT MODIFY ON	KEYSPACE linagora_keyspace TO linagora; // Permission to update data in tables on the appointed keyspace
+```
+
+**Warning**: The granted role doesn't have the right to create keyspaces, 
+thus, if you haven't created the keyspace, James server fails to start 
+is expected.  
+
+**Tips**
+
+Since all of Cassandra roles used by different James are supposed to 
+have a same set of permissions, you can reduce the works by creating a 
+base role set like `typical_james_role` with all of necessary permissions. 
+After that, with each James, just create a new role and grant the `typical_james_role` 
+to the newly created one. Note that, once a base role set is updated ( 
+granting or revoking rights) all granted roles are automatically updated.  
+
+E.g.
```

Comment:
CR

Reply:
I need line breaks, It makes the quote easier to read

Reply:
Sure, but I wonder if you need one more after